provider/postgis/postgis.go

```diff
@@ -199,14 +212,54 @@ func NewTileProvider(config dict.Dicter) (provider.Tiler, error) {
 		return nil, err
 	}
 
+	auto := DefaultAuto
```

Comment:
Can this be removed and we just use the layer count for determining if we're in auto mode or not. (line 222)